### DISCUSSION TITLE
Seperate the Event badges from Vanity badges

### DISCUSSION THREAD

**Discussion by romeogiggletoes** [Chatterino/chatterino2#3461] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Essentially asking for a separate 'Event' badges setting for badges such as GlitchCon, TwitchCon, GLHF Pledge, etc. I dislike how they are combined with gift, bits, and prime badges.
![2022_01_03_18_11_18_Chatterino_Settings](https://user-images.githubusercontent.com/97059214/147990278-cd67c87f-7ad4-457d-ad0b-b3e5fefbb27b.png)

### OS and Chatterino Version

Chatterino 2.3.4 (commit 4e5170799) on Windows 10

**Comment by Felanbird** [comment]
They are in the same slot on Twitch's side

**Comment by romeogiggletoes** [comment]
As an alternative, would there be no way to disable individual badges like how FFZ does for browser extension?

**Comment by ALazyMeme** [comment]
This is a suggestion, not an issue. Please carefully read and follow the issue templates - which would've lead you to creating a discussion:
![image](https://user-images.githubusercontent.com/12804673/148039950-c92471a9-93f5-41b9-b4a0-170e9527bab4.png)
